models/marts/history_assets.yml

```diff
@@ -9,9 +9,17 @@ models:
           field: cast(batch_run_date as timestamp)
           interval: 2
           config:
-            severity: warn
+            severity: error
           meta:
             description: "Monitors the freshness of your table over time, as the expected time between data updates."
+      - incremental_unique_combination_of_columns:
+          combination_of_columns:
+            - batch_run_date
```

Comment:
I'm not sure `batch_run_date` should be included. My assumption was that the `history_assets` table was unique on `asset_code, asset_issuer, and asset_type` otherwise we would have "duplicate assets" based where each asset would have multiple `batch_run_dates`

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/83/commits/6bd6abf411f043b3e30af1bfa1780741b9d9cac1